### PR TITLE
[GH-994] Fix WGS input preparation

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-994
- See: https://github.com/broadinstitute/dsde-pipelines/pull/1084/files?file-filters%5B%5D=.wdl#r470019578
- Also see: https://broadinstitute.slack.com/archives/CESEYJW9W/p1601306420061600

### Changes

- Fix WGS input preparation, see [here](https://broadinstitute.slack.com/archives/CESEYJW9W/p1600973191017200?thread_ts=1600968869.015600&cid=CESEYJW9W)

### Review Instructions

- No instructions.

### CODE REVIEW

File: api/src/wfl/module/wgs.clj [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 (def workflow-wdl
   "The top-level WDL file and its version."
-  {:release "ExternalWholeGenomeReprocessing_v1.0"
+  {:release "ExternalWholeGenomeReprocessing_v1.1.0"
```
Comment (rexwangcc) [inline api/src/wfl/module/wgs.clj:23]: This version is released to WARP but not dsde-pipelines yet.

**Review by jack-r-warren — APPROVED** [review]
This looks good but are we rolling this into [GH-983](https://broadinstitute.atlassian.net/browse/GH-983)?

**Comment by rexwangcc** [comment]
> This looks good but are we rolling this into [GH-983](https://broadinstitute.atlassian.net/browse/GH-983)?

Yeah, I think we should not merge this until the WARP transition is done!

**Comment by rexwangcc** [comment]
Close in favor of #191

**Pull request closed without merging**